packages/core/strapi/lib/services/server/index.js

```diff
@@ -81,6 +83,10 @@ const createServer = strapi => {
     },
 
     mount() {
+      if (state.mounted) {
```

Comment:
Shouldn't we throw instead of silently passing through? Or at least issue a warn message? It would facilitate spotting code that can lead to other issues (it doesn't feel normal to mount the app two times)

Reply:
And if at times we need to ensure that the app is mounted, shouldn't we priviligate checks such as a ensureMount() that does the `if !mounted -> mount()`

Reply:
I doesn't seem important as it's internal code and I needed to call mount I'll refactor it as it's not not needed anyway